bedrock/firefox/templates/firefox/os/index.html

```diff
@@ -1,119 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
- # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
-
-{% from "macros.html" import fxosnav with context %}
-
-{% extends "firefox/base-resp.html" %}
-
-{% block page_title_prefix %}{% endblock %}
-{% block page_image %}{{ static('img/firefox/os/page-image.png') }}{% endblock %}
-
-{% block page_title %}
-  {{_('Firefox OS — Just what you need — Great smartphone features, apps and more')}}
-{% endblock %}
-
-{% block page_desc %}
-  {{_('Firefox OS is designed to give you just what you need, just when you need it. It comes with apps for all you do and you can download more of your favorites from the Firefox Marketplace. Smart has never been so simple.')}}
-{% endblock %}
-
-{#- This will appear as <meta property="og:title"> which can be used for social share -#}
-{% block page_og_title %}
-  {{_('Firefox OS')}}
-{% endblock %}
-
-{#- This will appear as <meta property="og:description"> which can be used for social share -#}
-{% block page_og_desc %}
-  {{_('Designed to give you just what you need in a smartphone, with apps for all you do and more. Smart has never been so simple.')}}
-{% endblock %}
-
-{#- Override <meta property="og:url"> for social share -#}
-{% block page_og_url %}{{ url('firefox.os.index') }}{% endblock %}
-
-{% block body_id %}firefox-os{% endblock %}
-{% block body_class %}firefox-os v2-5{% endblock %}
-
-{% block page_css %}
-  {% stylesheet 'firefox_os_2_5' %}
-{% endblock %}
-
-{% block site_header %}{% endblock %}
-
-{% block site_header_unwrapped %}
-  {% call fxosnav('fxos') %}
-    <div id="cta-sticky">
-      <a href="#email-form-wrapper" class="primary-cta-signup cta-button-small newsletter-signup-toggle hidden" role="button" aria-haspopup="true">{{ _('Sign Up Now') }}</a>
-      <a href="#get-device" class="primary-cta-phone cta-button-small hidden" role="button" aria-haspopup="true">{{_('Get a phone')}}</a>
-    </div>
-  {% endcall %}
-{% endblock %}
-
-{% block content %}
-  <main role="main">
-    <section class="fxos-hero-space">
-      <div class="content-wrapper">
-        <div class="hero-content">
-          <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
-          <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you use Firefox OS on your Android device.')}}</p>
-          <div class="primary-cta">
-            <a href="{{ settings.B2G_DROID_URL }}" class="cta-button button button-green" data-pixel="52161" data-id="Mozilla_B2G_Activity_Tag_1">{{_('Get the Android App')}}</a>
```

Comment:
I believe you can delete `settings.B2G_DROID_URL` from `bedrock/settings/base.py`